docs/entities/servicecatalog.md

```diff
@@ -30,5 +30,12 @@ The Catalog will also be removed from the matching application namepsaces.
 
 ### Update
 
-When a registered service is being updated the Service Catalog will get updated too.
-The updated Sevice Catalog will be made available to all the application namespaces of matching Cluster Environment by Primaza.
+When a registered service is being updated, the appropriate service catalog will get updated too.
+The updated service catalog will be made available to all the application namespaces of matching cluster Eenvironment by Primaza.
```

Comment:
```suggestion
The updated Service Catalog will be made available to all the application namespaces of matching Cluster Environment by Primaza.
```

Reply:
OK

Reply:
made the change with the typo, but should be consistent with Upper Case for Resource name vs lower case. For example Service Catalog vs service catalog or Registered Service vs registered service. I think we should stick with lower case unless you are referring to the K8 name which will be ServiceCatalog and RegisteredService.